appointment/forms.py

```diff
@@ -0,0 +1,23 @@
+from django import forms
+from .models import Meeting
+
+
+class DateInput(forms.DateInput):
+    """Datetime picker type."""
+    input_type = 'datetime-local'
```

Comment:
Use for use date-picker in start-time, end-time. ( firefox, safari not supported )